package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@across-protocol/sdk",
   "author": "UMA Team",
-  "version": "4.3.12",
+  "version": "4.3.13-alpha.0",
```

Comment:
are you going to change this before merge?

Reply:
Just updated here. I wanted to make sure that this worked as expected in the relayer. https://github.com/across-protocol/sdk/pull/1128/commits/7ab863ee76019ec7672e2038b4ed1ddd831ab6a0